.github/workflows/ignore-license-checker.yml

```diff
@@ -0,0 +1,11 @@
+name: license checker
+on:
+  pull_request:
+    types: [opened, synchronize, edited, reopened]
+    paths-ignore:
+      - '**/package.json'
+jobs:
+  build:
```

Comment:
@htztomic job name should be license-checker ?

Reply:
@insignias updated